tests/robot/COMPOSITION_TESTS/CREATE_COMPOSITION/B.6.c_alternative_flow_2_create_persistent_COMPOSITION_for_the_same_opt_twice.robot

```diff
@@ -29,23 +29,23 @@ Force Tags      125    future
 
 
 *** Test Cases ***
-Alternative flow 2 create persistent COMPOSITION for the same archetype twice RAW_JSON
+Alternative flow 2 create persistent COMPOSITION for the same opt twice RAW_JSON
```

Comment:
@wlad wouldn't be nice to reference the numbering in the doc? in this case B.6.c or having the link as a comment https://github.com/ehrbase/ehrbase/blob/develop/doc/conformance_testing/EHR_COMPOSITION.md#b6c-alternative-flow-2-create-persistent-composition-for-the-same-opt-twice ?